support-frontend/app/services/mparticle/MParticleTokenProvider.scala

```diff
@@ -63,6 +63,7 @@ class MParticleTokenProvider(
 
   override val wsUrl: String = config.tokenUrl
   override val verboseLogging: Boolean = false
+  override val requestLogging: Boolean = false
```

Comment:
I've introduced this parameter to the `WebServiceHelper` because we do not want to log the request body when fetching the oauth token